hal/src/mrc/include/mrc/NetComm.h

```diff
@@ -16,56 +18,94 @@
 
 namespace mrc {
 
-struct OpModeTrace {
-  uint64_t Hash{0};
-  bool Enabled{false};
-  bool NoCurrentOpMpode{false};
-
-  static OpModeTrace FromValue(uint64_t Value) {
-    return OpModeTrace{
-        .Hash = Value & 0x3FFFFFFFFFFFFFFF,
-        .Enabled = (Value & 0x8000000000000000) == 0,
-        .NoCurrentOpMpode = (Value & 0x4000000000000000) != 0,
-    };
+struct OpModeHash {
+  uint64_t Hash : 56 {};
+  uint64_t IsAuto : 1 {};
+  uint64_t IsTest : 1 {};
+  uint64_t IsEnabled : 1 {};
+  uint64_t Reserved : 6 {};
```

Comment:
Personally feels a little weird that IsAuto, IsTest, and IsEnabled are before Reserved when `FromValue()` and `ToValue()` put them at the most significant 3 bits, but that might just be me. (Feel free to resolve this conversation if this is fine)

Reply:
Another bug, I thought they were the low 3 bits. Fixing.